platform/ontology/client.go

```diff
@@ -17,3 +63,50 @@ func (c *Client) GetTxsOfAddress(address, assetName string) (txPage *TxPage, err
 	err = c.Get(&txPage, url, nil)
 	return
 }
+
+func (c *Client) CurrentBlockNumber() (int64, error) {
+	url := fmt.Sprintf("blocklist/%d", 1)
+	var (
+		response blockListResponse
+		height   int64
+	)
+	err := c.Get(&response, url, nil)
+	if err != nil {
+		return 0, err
+	}
+	if len(response.Result) > 0 {
+		height = (int64)(response.Result[0].Height)
+	}
+	return height, nil
+}
+
+func (c *Client) GetBlockByNumber(num int64) (*blockatlas.Block, error) {
+	url := fmt.Sprintf("block/%d", num)
+	var (
+		response blockByNumberResponse
+		block    blockatlas.Block
+		txs      []blockatlas.Tx
+	)
+	err := c.Get(&response, url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	if response.Error == 0 {
```

Comment:
please, avoid parsing inside the client methods. The file should need to be responsible only for client methods, do you need to do that inside the `api.go` file